Tests/SwiftFormatPrettyPrintTests/GuardStmtTests.swift

```diff
@@ -47,4 +47,28 @@ public class GuardStmtTests: PrettyPrintTestCase {
 
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 35)
   }
+
+  public func testGuardWithFuncCall() {
+    let input =
+      """
+      guard let myvar = myClass.itsFunc(first: .someStuff, second: .moreStuff).first else {
```

Comment:
Can you also add a second test with two long function calls? Based on the change above, I think we'd want this:

```swift
guard let myvar1 = myClass.itsFunc(first: .someStuff, second: .moreStuff).first, let myvar2 = myClass.otherFunc(first: .someStuff, second: .moreStuff).last else {
  // do stuff
}
```

to format like this:

```swift
guard
  let myvar1 = myClass.itsFunc(
    first: .someStuff,
    second: .moreStuff
  ).first,
  let myvar2 = myClass.otherFunc(
    first: .someStuff,
    second: .moreStuff
  ).last
else {
  // do stuff
}
```

So it'll be good to verify it and make sure it keeps working.

Reply:
Added the extra test - it passes.